Kkmattil braker 1 (#1352)
CSCfi/csc-user-guide: docs/apps/braker.md

> # BAKER

Reviewer: ```suggestion
# BRAKER
```

## Description

> Braker is a tool for eukaryotic genome annotation. 

Reviewer: ```suggestion
BRAKER is a tool for eukaryotic genome annotation. 
```

It uses genomic and RNA-Seq data to automatically generate full gene structure annotations in novel genome.

> Barker is based on GeneMark-ET R2 and AUGUSTUS pipelines.

Reviewer: ```suggestion
BRAKER is based on GeneMark-ET R2 and AUGUSTUS pipelines.
```

## License

Free to use and open source under [Artistic License] (https://opensource.org/licenses/artistic-license-1.0)

## Version

Version on CSC's Servers

Puhti: 2.1.6

## Usage

> In Puhti Braker should be used only in batch jobs. Either in normal batch jobs or in interactive batch jobs.

Reviewer: ```suggestion
In Puhti BRAKER should be used only in batch jobs. Either in normal batch jobs or in interactive batch jobs.
```

You can start interative batch job with command:

```text
sinteractive -i
```

> In batch job, you can initialize Braker environment with command

Reviewer: ```suggestion
In batch job, you can initialize BRAKER environment with command
```

```text
module load braker
```

> After that you can launch a Braker job with command:

Reviewer: ```suggestion
After that you can launch a BRAKER job with command:
```

```text
braker-puhti
```

> This command should be used in stead of the original _braker.pl_ script as it automatically do 

Reviewer: ```suggestion
This command should be used in stead of the original _braker.pl_ script, as it automatically sets 
```

> some settings that allow you to run baraker. _braker-puhti_ is able to use all the command line options

Reviewer: ```suggestion
some parameters that allow you to run BRAKER. _braker-puhti_ is able to use all the command line options
```

of _braker.pl_. To see the options, run command:

```text
braker-puhti --help
```

> Sample braker command in Puhti:

Reviewer: ```suggestion
Sample BRAKER command in Puhti:
```
